lusidtools/pandas_utils/lusid_pandas.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+from flatten_json import flatten
+
+
+def lusid_respose_to_pd(lusid_response):
+
+    """
+    This function takes a LUSID API response and attempts to convert the response into a Pandas DataFrame or Series.
+    Not all LUSID API responses are the same datatype.
```

Comment:
List out the types of supported responses e.g. object containing `values`, dictionary etc.

Reply:
Thanks. Corrected in latest commit.